### PR TITLE
Added test for the Symfony constraints

### PR DESCRIPTION
**Pull request by martijnvankempen**
Added tests to see if the Symfony constraints are processed successfully

### CODE REVIEW

File: Tests/Functional/Entity/SymfonyConstraints.php [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+     * @Serializer\SerializedName("propertyRegex")
+     *
+     * @Assert\Regex(pattern="/[a-z]{2}/")
+     */
+    private $propertyRegex;
+
+    /**
+     * @Serializer\Type("integer")
+     * @Serializer\Expose
+     * @Serializer\SerializedName("propertyCount")
```
Comment (GuilhemN) [inline Tests/Functional/Entity/SymfonyConstraints.php]: I'd rather test this in the normal `FunctionalTest` class to use symfony's serializer and be able to remove jms annotations.

Comment (martijnvankempen) [reply Tests/Functional/Entity/SymfonyConstraints.php]: @GuilhemN I'm getting the following error when I remove the jms annotations:
"Uncaught PHP Exception PHPUnit\Framework\Error\Warning: "count(): Parameter must be an array or an object that implements Countable"

File: Tests/Functional/Controller/JMSController.php [hunk 2]
```diff
@@ -32,6 +33,18 @@ public function userAction()
...
 
+    /**
+     * @Route("/api/symfonyConstraints")
+     * @SWG\Response(
+     *     response=200,
+     *     description="Success",
+     *     @Model(type=SymfonyConstraints::class)
+     * )
+     */
+    public function symfonyConstraintsAction()
```
Comment (GuilhemN) [inline Tests/Functional/Controller/JMSController.php]: This route can be removed, right ?

**Comment by GuilhemN** [comment]
Thank you @martijnvankempen! :)

**Pull request merged**